packages/cosmwasm-stargate/src/cosmwasmclient.searchtx.spec.ts

```diff
@@ -8,7 +8,8 @@ import {
   makeSignDoc,
   Registry,
 } from "@cosmjs/proto-signing";
-import { BroadcastTxResponse, codec, isBroadcastTxFailure, isBroadcastTxSuccess } from "@cosmjs/stargate";
+import { BroadcastTxResponse, isBroadcastTxFailure, isBroadcastTxSuccess } from "@cosmjs/stargate";
+import { Tx, TxRaw } from "@cosmjs/stargate/build/codec/cosmos/tx/v1beta1/tx";
```

Comment:
Okay, this is really ugly. I thought the `"main": "build/index.js"` would define the base path for `@cosmjs/stargate`, which is why I thought we get `@cosmjs/stargate/codec/cosmos/tx/v1beta1/tx`. But that's not the case.

I wonder if this is still out problem, if we re-export the types in `@cosmjs/stargate` that are part of the package's API. But if I recall correctly this is already too many to be maintainable.